bin/oneoff/update_pd_applications_with_new_capitalization.rb

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+
+puts "Updating principal approval applications...\n\n"
+# Update capitalization on each principal approval application and save it,
+# then update, save, and score its corresponding teacher application
+Pd::Application::PrincipalApproval1819Application.find_each do |principal_application|
+  if principal_application.form_data.include?("(please explain)")
+    principal_application.form_data = principal_application.form_data.gsub("(please explain)", "(Please Explain)")
+    principal_application.save
+    if principal_application.errors
+      puts "principal app: #{principal_application.id} errors: #{principal_application.errors.full_messages}"
```

Comment:
If you have an error here, consider not doing anything else with this application.